backend/apps/checks.py

```diff
@@ -92,19 +93,42 @@ def check_slack_webhook(app_configs=None, **kwargs):
     return errors
 
 
-@register_check(Tags.env_tag, [Environment.DEVELOPMENT, Environment.PRODUCTION], exclude_if_ci=True)
-def check_mapbox_token(app_configs=None, **kwargs):
+@register_check(
+    Tags.mail_tag, [Environment.DEVELOPMENT, Environment.PRODUCTION], exclude_if_ci=True
+)
+def check_map_settings(app_configs=None, **kwargs):
     errors = []
 
-    if settings.MAPBOX_TOKEN is None:
+    if settings.MAPVIEW_BACKEND == "open_street_map":
+        errors.append(
+            Warning(
+                "Usage restricted tile server.",
+                hint="You are using an open street map tile server for viewing the map, which is subject to usage "
+                "restrictions. \nIf you plan to put this system into production, please consider setting up "
+                "your own tile server or using a commercial service, e.g. mapbox."
+                "We readily provide an integration for mapbox, which you can use by setting MAPVIEW_BACKEND='mapbox'"
+                " and adding your API key via 'export MAPBOX_TOKEN=<<yourToken>>'",
+                id="map.W001",
+            )
+        )
+    elif settings.MAPVIEW_BACKEND == "mapbox":
+        if settings.MAPBOX_TOKEN is None:
+            errors.append(
+                Error(
+                    "Mapbox token not found.",
+                    hint=(
+                        "You have to set the Mapbox token in you environment with "
+                        "'export MAPBOX_TOKEN=<<yourToken>>'."
```

Comment:
```suggestion
                        "'export MAPBOX_TOKEN=<<yourToken>>'. or if you use Docker in the corresponding env file"
```

Reply:
Either everywhere or nowhere. I'd suggest changing the wording to "set the environment Variable MAPBOX_TOKEN"

Reply:
I mean we should change this in all checks and remove "export ....=...." because that's bash specific. It won't work neither with docker nor Windows env for example. We should keep the error message environment-agnostic.

Reply:
Removed all exports as well

Reply:
@maltezacharias @feeds I would mention it in the help texts because the only reason for the texts is ease of use and it's not obvious that "set it in your environment" doesn't have any effect when someone uses Docker (which is (with an easy setup from our side) the most likely option if someone just wants to try it out)

Reply:
I'm not sure I understand this correctly. Can you maybe make a commit suggestion?